spec.html

```diff
@@ -212,6 +212,47 @@ <h1>
   </emu-clause>
 </emu-clause>
 
+<emu-clause id="sec-ordinary-and-exotic-objects-behaviours">
+  <h1>Ordinary and Exotic Objects Behaviours</h1>
+  <emu-clause id="sec-built-in-function-objects">
+    <emu-clause id="sec-createbuiltinfunction" type="abstract operation">
+      <h1>
+        CreateBuiltinFunction (
+          _behaviour_: an Abstract Closure, a set of algorithm steps, or some other definition of a function's behaviour provided in this specification,
+          _length_: a non-negative integer or +&infin;,
+          _name_: a property key or a Private Name,
+          _additionalInternalSlotsList_: a List of names of internal slots,
+          optional _realm_: a Realm Record,
+          optional _prototype_: an Object or *null*,
+          optional _prefix_: a String,
+        ): a function object
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>_additionalInternalSlotsList_ contains the names of additional internal slots that must be defined as part of the object. This operation creates a built-in function object.</dd>
+      </dl>
+      <emu-alg>
+        1. If _realm_ is not present, set _realm_ to the current Realm Record.
+        1. If _prototype_ is not present, set _prototype_ to _realm_.[[Intrinsics]].[[%Function.prototype%]].
+        1. Let _internalSlotsList_ be a List containing the names of all the internal slots that <emu-xref href="#sec-built-in-function-objects"></emu-xref> requires for the built-in function object that is about to be created.
+        1. Append to _internalSlotsList_ the elements of _additionalInternalSlotsList_.
+        1. Let _func_ be a new built-in function object that, when called, performs the action described by _behaviour_ using the provided arguments as the values of the corresponding <del>parameters</del><ins>required parameters, optional parameters, and rest parameters</ins> specified by _behaviour_. The new function object has internal slots whose names are the elements of _internalSlotsList_, and an [[InitialName]] internal slot.
```

Comment:
/cc @michaelficarra

Reply:
Do we really need this change? The [definition of abstract closures](https://tc39.es/ecma262/#sec-abstract-closure) is already kinda vague about how they accept parameters anyway.

Reply:
We'll discuss it in the editor call today. @jmdyck seemed to think it was necessary. You're all welcome to attend.

Reply:
We could maybe say something like "using the provided arguments to set the values of _behaviour_'s parameters, as described in [Algorithm Conventions]"

Reply:
I'm happy to defer to you all here; I'm glad the editors are working on this.